amazon-kinesis-client-multilang/src/test/java/software/amazon/kinesis/multilang/MultiLangDaemonConfigTest.java

```diff
@@ -48,39 +58,170 @@ public class MultiLangDaemonConfigTest {
     @Mock
     private KinesisClientLibConfigurator configurator;
```

Comment:
This is not a `@Mock` when it's instantiated on line ~85.

Code should not be misleading.